src/pages/homeCollection/kitAssembly.js

```diff
@@ -110,11 +110,14 @@ const enableEnterKeystroke = () => {
 
 const performQCcheck = (inputBox2, inputBox1, errorTag, errorMsg) => {
   const checkInputBox2 = document.getElementById(inputBox2);
+  if(errorMsg == "") {
+    document.getElementById(errorTag).innerHTML = `` 
+  }
   if (checkInputBox2) {
     checkInputBox2.addEventListener("input", (e) => {
       const checkInputBox2Value = e.target.value.trim();
       const checkInputBox1Value = document.getElementById(inputBox1).value.trim();
-      if (checkInputBox2Value !== checkInputBox1Value) {
+      if (checkInputBox2Value != checkInputBox1Value) {
```

Comment:
```suggestion
      if (checkInputBox2Value !== checkInputBox1Value) {
```

The strict equality operator check may need to be added back.